### PR TITLE
Issue510 durack1 update durack subpages

### PR DESCRIPTION
**Pull request by durack1**
Ref #510

### CODE REVIEW

**Comment by durack1** [comment]
Fix #510 

@mauzey1 please review and merge if all checks out

**Comment by durack1** [comment]
@mauzey1 these are really standalone "subpages" so no need to be linked off other pages, but as these exist, we could reconsider in the future

File: staff/durack/subpages/salinity/index.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                    (CSIRO: Alicia Navidad) - Clicking <a
+                        href="tif/120410_Argo_SouthernOceanDeployment.tif"><strong>HERE</strong></a> will download a
+                    high-resolution *.tif file suitable for print.</p>
+            </div> <!--id=content-->
+
+            <div id="content" class="index">
+                <img style="padding: 5px; border: thin solid white;" src="png/120412_GlobalSalinityMP4.png" alt="CSIRO"
+                    align="left">
+                <p>Video 1. Selected footage of oceanographic videos for use in broadcast (CSIRO) - Clicking <a
+                        href="http://mbf.cc/xFF6f"><strong>HERE (783Mb)</strong></a> will download a high-definition
```
Comment (mauzey1) [inline staff/durack/subpages/salinity/index.html:124]: @durack1 These mbf.cc links are not working.  Do you have these files?  We could host them on our server instead.

**Review by mauzey1 — CHANGES_REQUESTED** [review]

File: staff/durack/subpages/salinity/index.html (hunk 1 above)
Comment (durack1) [reply staff/durack/subpages/salinity/index.html:124]: @mauzey1 thanks for checking, I just took a scour and don't seem to have downloaded these files to my personal archive - I wonder if these are in a backup somewhere, doubtful if they are not close to the directory

Comment (durack1) [reply staff/durack/subpages/salinity/index.html:124]: @mauzey1 just found a [youtube video](https://www.youtube.com/watch?v=odG83ruW33Q) which should be a reasonable switchout - I'll tweak this PR and ping you when it's in place this afternoon

**Comment by durack1** [comment]
@mauzey1 link switched out - this should be good for review and merge

**Review by mauzey1 — APPROVED** [review]

**Pull request merged**